hadoop-hdds/docs/content/concept/OzoneManager.zh.md

```diff
@@ -64,10 +64,60 @@ OM 使用 Apache Ratis（Raft 协议的一种实现）来复制 OM 的状态，
 * 数据写入完成后，用户会更新该块在 OM 中的信息。
 
 
-### 读取键
+### 读取Key
 
 * 键读取相对比较简单，用户首先向 OM 请求该键的块列表。
 
 * OM 返回块列表以及对应的块 token。
 
-* 用户连接数据节点，出示块 token，然后读取键数据。
\ No newline at end of file
+* 用户连接数据节点，出示块 token，然后读取键数据。
+
+
+## Ozone Manager的主要组件
+
+为了详细地了解Ozone Manager，本节针对它所提供的网络服务和持久化状态提供一个快速概述。
+
+Ozone为客户端和管理命令提供网络服务，主要的服务如下：
+
+ * Key, Bucket, Volume / 增删改查
+ * 分块上传 (Initiate, Complete…)
+    * 支持大文件的多步上传
+ * 与FS相关的调用 (针对分层查询而非对象存储的扁平化命名空间进行了优化)
+   * GetFileStatus, CreateDirectory, CreateFile, LookupFile
+ * ACL 相关
+   * 使用 [内部 ACLs]({{< ref "security/SecurityAcls.md" >}}) 代替 [Ranger]({{< ref "security/SecurityWithRanger.md" >}}) 管理访问控制列表
+ * Delegation token (获取 / 刷新 / 取消)
+   * 用于安全相关
+ * 管理API
+   * 获取S3的访问密钥
+   * ServiceList (用于服务发现)
+   * DBUpdates (用于 [Recon]({{< ref "feature/Recon.md" >}}) 下载快照)
```

Comment:
```suggestion
   * DBUpdates (用于 [Recon]({{< ref path="feature/Recon.md" lang="en" >}}) 下载快照)
```
To reference documents for different languages, you need to configure the "lang" parameter.

Reply:
![image](https://user-images.githubusercontent.com/14295594/98646106-db256300-236d-11eb-936a-35647a6e66d3.png)